### PR TITLE
DROOLS-4357 : Two download buttons are in Guided Decision Tables

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-4357

https://github.com/kiegroup/appformer/pull/745
https://github.com/kiegroup/kie-wb-common/pull/2777
https://github.com/kiegroup/drools-wb/pull/1196

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableEditorPresenterTest.java [hunk 1]
```diff
@@ -132,6 +132,14 @@ public void testSetupMenuBar() {
...
     }
 
+    @Test
+    public void testDownloads() {
+        presenter.makeMenuBar();
+        presenter.makeMenuBar();
+
+        verify(downloadMenuItemBuilder).build(any());
+    }
+
```
Comment (Rikkola) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableEditorPresenterTest.java]: Fix in kie-wb-common, but the test is here due to overwrites.

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableEditorPresenter.java [hunk 2]
```diff
@@ -203,7 +203,7 @@ public void getMenus(final Consumer<Menus> menusConsumer) {
         super.getMenus(menusConsumer);
     }
 
-    @Override
+    @Override
```
Comment (Rikkola) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/GuidedDecisionTableEditorPresenter.java]: I'll clean this up.

**Comment by Bonuseto** [comment]
jenkins execute full downstream build

**Comment by ederign** [comment]
Seems like fails are not related.

**Pull request merged**